### PR TITLE
PAT-1863 Update DCS Screen Query Result to list POMs

### PR DESCRIPTION
**Pull request by steverendell**
throttle POM api calls

### CODE REVIEW

File: integration-tests/build.gradle [hunk 1]
```diff
@@ -3,14 +3,9 @@ plugins {
 }
 
 ext {
```
Comment (steverendell) [inline integration-tests/build.gradle:5]: tidy up a bit

File: server/services/offendersService.js [hunk 2]
```diff
@@ -76,6 +76,23 @@ async function getOffenceMap(offenderList, nomisClient) {
   return new Map(offences.map(offence => [offence.bookingId, offence]))
 }
```
Comment (steverendell) [inline server/services/offendersService.js:78]: uses same technique as PF batch

File: .circleci/config.yml [hunk 3]
```diff
@@ -2,8 +2,7 @@ version: 2.1
 
 orbs:
   hmpps: ministryofjustice/hmpps@3.11
```
Comment (steverendell) [inline .circleci/config.yml:4]: applying update that is in PF

File: package-lock.json [hunk 4]
```diff
@@ -8338,9 +8338,9 @@
       }
     },
     "node_modules/tar": {
```
Comment (steverendell) [inline package-lock.json:8340]: security vulnerability

**Review by lbennett-moj — APPROVED** [review]

**Pull request merged**